### ISSUE TITLE
Can `UnionFind::alloc` be used to add elements after `UnionFind::union` has been called?

### ISSUE THREAD

**Issue by sebschmi** [tov/disjoint-sets-rs#7]
The documentation does not mention if `UnionFind::alloc` behaves correctly if the union find data structure has been altered already by calling `UnionFind::union` after construction. Is this the case? Or can `UnionFind::alloc` only be used before the first call to `UnionFind::union`?

**Comment by tov** [comment]
Yes, that should work correctly. `UnionFind::alloc` creates new singleton sets, but it doesn't change the existing sets. Perhaps the documentation should say this.

**Comment by sebschmi** [comment]
Nice, thanks! Yeah I think it would be good to point it out in the documentation, it is a useful feature.
